bound/golang/web5c/export.go

```diff
@@ -0,0 +1,71 @@
+package web5c
+
+/*
+#include <stdlib.h>
+
+#include "../../../bindings/web5_c/web5_c.h"
+*/
```

Comment:
```suggestion
```

Reply:
this is required, it's the way cgo works, it's called the "preamble"